tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/invocations/FuncInvocationExprTest.java

```diff
@@ -195,6 +195,16 @@ public void testFunctionCallNegativeCases() {
                 "incompatible types: expected 'int[]', found '(int|string)[3]'", 73, 29);
         validateError(funcInvocationNegative, i++,
                 "incompatible types: expected 'int[]', found 'anydata[]'", 74, 29);
+        validateError(funcInvocationNegative, i++,
+                "too many arguments in call to 'sum()'", 102, 18);
```

Comment:
This seems not right to me. The actual error is we don't accept the name arguments in this function. We should give a proper error for this. Can we fix it in the same PR?

Reply:
Let's create an issue for the mentioned improvement and send it as a separate PR. I will merge this PR since this is passed checks.

Reply:
Created in #39818